PathsActivity.py

```diff
@@ -73,20 +74,22 @@ def __init__(self, handle):
         else:
             self.colors = ['#A0FFA0', '#FF8080']
 
-        self._setup_toolbars()
-        self._setup_dispatch_table()
-
         # Create a canvas
         canvas = Gtk.DrawingArea()
-        canvas.set_size_request(Gdk.Screen.width(), Gdk.Screen.height())
+        canvas.set_size_request(Gdk.Screen.width(), Gdk.Screen.height() - style.GRID_CELL_SIZE)
         self.set_canvas(canvas)
         canvas.show()
-        self.show_all()
 
         self._game = Game(canvas, parent=self, colors=self.colors)
+
+        self._setup_toolbars()
+        self._setup_dispatch_table()
+
+
```

Comment:
And even the reordering doesnot make any functional difference

Reply:
Reordering does make functional difference in theory. See `sugargame` activity to see how its important